k8s/events-monitor/prod-events-notifs-cronjob.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
 spec:
   successfulJobsHistoryLimit: 1
   failedJobsHistoryLimit: 1
-  schedule: "@hourly"
+  schedule: "30 * * * *"
```

Comment:
Changed the schedule to 30 minutes pasted the hour. @Baalmart

Reply:
Awesome, thanks @NoahNsimbe